src/ERC20DelegatesUpgradeable.sol

```diff
@@ -1,24 +1,23 @@
-// SPDX-License-Identifier: MIT
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
why?

Reply:
it's the licence of all of our code, except the IRM (MIT) and Blue (BSL)

Reply:
See [conversation above ](https://github.com/morpho-org/morpho-token-upgradeable/pull/29#discussion_r1810055200), feel free to re-open that one above